pkg/cli/clisqlshell/sql.go

```diff
@@ -2209,65 +2201,49 @@ func (c *cliState) maybeHandleInterrupt() func() {
 	signal.Notify(intCh, os.Interrupt)
 	ctx, cancel := context.WithCancel(context.Background())
 
-	if !queryCancelEnabled {
-		go func() {
-			for {
-				select {
-				case <-ctx.Done():
-					// Shell is terminating.
-					return
-				case <-intCh:
+	go func() {
+		for {
```

Comment:
was there a test for this functionality somewhere?

Reply:
done